src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -370,7 +377,7 @@ private void TrySetError(SyntaxDiagnosticInfo error)
                 Error ??= error;
             }
 
-            internal void ScanInterpolatedStringLiteralTop(
+            internal void ScanStringLiteralTop(
```

Comment:
I didn't follow why 'Interpolated' was deleted from this name. Wouldn't it make sense to use 'InterpolatedOrRaw' like many of the other methods are doing? Or did I miss something?

Reply:
Sure!  I can def rename.